src/components/HomepageFeatures.js

```diff
@@ -9,7 +9,7 @@ const FeatureList = [
     description: (
       <>
         The syntax of Unv language is inspired by Python. So It's nearly similar to English.
-        instead of too many brackets ot depends on whitespace.
+        instead of too many brackets or depending on whitespace.
```

Comment:
```suggestion
        Instead of too many brackets, it depends on whitespaces.
```

That's what I wanted to write. But feel free to suggest edits.

Reply:
@Pandademic ^ above changes

Reply:
Oh I didn't see that , It's a lot better